src/coreclr/tests/src/baseservices/compilerservices/RuntimeHelpers/AllocateTypeAssociatedMemory.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+using System;
+using TestLibrary;
+using System.Runtime.CompilerServices;
+
+class AllocateTypeAssociatedMemoryTest
+{
+    private static void ValidateInvalidArguments()
```

Comment:
This is a regular API test. It would be better for it to live under src\libraries

Reply:
Do you have a suggestion on where to put this? I don't know what project to add this to and honestly don't really want to figure out how to add something new to the libraries partition.

Reply:
Sorry nevermind. I found it.

Reply:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/RuntimeHelpersTests.cs